OpenSim/Actuators/ActiveForceLengthCurve.cpp

```diff
@@ -158,6 +158,15 @@ double ActiveForceLengthCurve::calcValue(double normFiberLength) const
     return m_curve.calcValue(normFiberLength);
 }
 
+SmoothSegmentedFunction::ValueAndDerivative ActiveForceLengthCurve::
+    calcValueAndDerivative(double normFiberLength) const
+{
+    SimTK_ASSERT(
```

Comment:
There is now `OPENSIM_ASSERT` (see: https://github.com/opensim-org/opensim-core/pull/3548), which provides better user feedback (component names, etc.)